cmd/resim/cli.go

```diff
@@ -12,6 +13,7 @@ const EnvPrefix = "RESIM"
 func main() {
 	viper.SetEnvPrefix(EnvPrefix)
 	viper.AutomaticEnv()
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
```

Comment:
Is this replacement the right way round? This will replace existing `-` with `_`, I'd have thought (guessed...) that this should replace the `_` in an `ENV_VAR` with `-` so it can be compared to e.g. `project-id`

Reply:
So, the mapping goes in the other direction - it maps our CLI flag key to the ENV key.  I'll add a comment because this is probably going to bite others.